### PR TITLE
feat: WeekPicker support hover style

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    WeekPicker support hover style.       |
| 🇨🇳 Chinese |   WeekPicker 支持鼠标悬浮样式。      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/date-picker/style/index.ts [hunk 1]
```diff
@@ -739,11 +739,13 @@ export const genPanelStyle = (token: SharedPickerToken): CSSObject => {
             },
           },
 
-          '&:hover td': {
-            background: controlItemBgHover,
-          },
+          // '&:hover td': {
+          //   background: controlItemBgHover,
+          // },
```
Comment (kiner-tang) [inline components/date-picker/style/index.ts:744]: 如果没用了，要不直接删掉？
```suggestion
```

Comment (zombieJ) [reply components/date-picker/style/index.ts]: 嗯，忘删了。我删掉它

**Comment by MadCcc** [comment]
image
这个圆角是不是可以去掉

**Comment by afc163** [comment]
图片
单选样式没了。

**Review by li-jia-nan — APPROVED** [review]

**Comment by afc163** [comment]
还是不对，原来的 hover 背景色是灰的。

图片

现在变成高亮的了。

图片

**Pull request merged**